base/special/exp.jl

```diff
@@ -0,0 +1,137 @@
+# Based on FreeBSD lib/msun/src/e_exp.c
+# ====================================================
+# Copyright (C) 2004 by Sun Microsystems, Inc. All rights reserved. Permission
```

Comment:
You only did half of it.

> Is it necessary to add a line here https://github.com/JuliaLang/julia/blob/master/contrib/add_license_to_files.jl#L39 to skip this?

Yes. "exceptions to the license file and contrib/add_license_to_files.jl"